src/test/java/uk/gov/hmcts/cmc/claimstore/models/ClaimDataTest.java

```diff
@@ -131,11 +131,27 @@ public void shouldBeInvalidWhenGivenTooManyDefendants() {
             .addDefendant(SampleTheirDetails.builder().individualDetails())
             .addDefendant(SampleTheirDetails.builder().individualDetails())
             .addDefendant(SampleTheirDetails.builder().individualDetails())
+            .addDefendant(SampleTheirDetails.builder().individualDetails())
```

Comment:
um, is it possible to write a function which takes how many defendants you want as an argument?
This isn't ideal

Reply:
agree.

Reply:
Done